src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0; // Initialize the sum to zero
+
+  function adder(...args) {
```

Comment:
The `adder` function should only accept a single argument at a time according to the task description. The use of rest parameters `...args` allows multiple arguments, which is not in line with the specified behavior of the function. Consider revising the function to accept only one argument.